srcpkgs/zchunk/template

```diff
@@ -18,8 +18,20 @@ case "$XBPS_TARGET_MACHINE" in
 		;;
 esac
 
-post_install() {
-	rm -rf ${DESTDIR}/usr/include
-	rm -rf ${DESTDIR}/usr/lib/pkgconfig
-	vlicense LICENSE
+libzchunk_package() {
+	short_desk+=" - shared libraries"
+	pkg_install() {
+		vlicense LICENSE
+		vmove "usr/lib/*.so.*"
+	}
+}
+
+libzchunk-devel_package() {
+	short_desc+=" - development files"
+	depends="libzchunk-${version}_${revision}"
```

Comment:
depends on libressl-devel and libztd-devel, see `xpcdeps` from `xtools`

```
$ xpcdeps zck
libzstd-devel -> libzstd
libressl-devel -> openssl
```

Reply:
@maxice8 Fixed. Thanks for taking the time to review.